README.rst

```diff
@@ -119,25 +119,42 @@ The RGBDS source code file structure somewhat resembles the following:
 3. History
 ----------
 
-- 1997-07-03: Carsten Sørensen (a.k.a. SurfSmurf) writes ASMotor as a
-  general-purpose assembler/linker system for DOS/Win32.
+- 1996-10-01: Carsten Sørensen (a.k.a. SurfSmurf) releases
+  `xAsm <http://otakunozoku.com/RGBDSdocs/asm.htm>`__,
+  `xLink <http://otakunozoku.com/RGBDSdocs/link.htm>`__, and
+  `RGBFix <http://otakunozoku.com/RGBDSdocs/fix.htm>`__,
+  a Game Boy SM83 (GBZ80) assembler/linker system for DOS/Win32.
 
-- 1999-08-01: Justin Lloyd (a.k.a. Otaku no Zoku) adapts ASMotor to read and
-  produce GBZ80 assembly/machine code, and releases this version as RGBDS.
+- 1997-07-03: Sørensen releases `ASMotor <http://otakunozoku.com/RGBDSdocs/geninfo.htm>`__,
+  packaging the three programs together and moving towards making them a
+  general-purpose target-independent system.
+
+- 1999-08-01: Justin Lloyd (a.k.a. Otaku no Zoku) adapts ASMotor to re-focus
+  on SM83 assembly/machine code, and releases this version as
+  `RGBDS <http://otakunozoku.com/rednex-gameboy-development-system/>`__.
 
 - 2009-06-11: Vegard Nossum adapts the code to be more UNIX-like and releases
-  this version as rgbds-linux on `GitHub <https://github.com/vegard/rgbds-linux>`__.
+  this version as `rgbds-linux <https://github.com/vegard/rgbds-linux>`__.
+
+- 2010-01-12: Anthony J. Bentley `forks <https://github.com/bentley>`__ Nossum's
+  repository. The fork becomes the reference implementation of RGBDS.
+
+- 2015-01-18: stag019 begins implementing `rgbgfx <https://github.com/stag019/rgbgfx>`__,
+  a PNG‐to‐Game Boy graphics converter, for eventual integration into RGBDS.
 
-- 2010-01-12: Anthony J. Bentley forks that repository. The fork becomes the reference
-  implementation of RGBDS.
+- 2016-09-05: rgbgfx is `integrated <https://github.com/gbdev/rgbds/commit/c3c31138ddbd8680d4e67957e387f2816798a71b>`__
+  into Bentley's repository.
 
-- 2017-02-23: Bentley's repository is moved to a neutral name, Rednex.
+- 2017-02-23: Bentley's repository is moved to the `rednex <https://github.com/rednex>`__
+  organization.
 
-- 2018-01-26: The codebase relicensed under the MIT license.
+- 2018-01-26: The codebase is `relicensed <https://github.com/gbdev/rgbds/issues/128>`__
+  under the MIT license.
 
-- 2020-09-15: The repository is moved to the `gbdev <https://github.com/gbdev>`__
-  organization. The `rgbds.gbdev.io <https://rgbds.gbdev.io>`__ website serving documentation
-  and downloads is created.
+- 2020-09-15: The repository is `moved <https://github.com/gbdev/rgbds/issues/567>`__
+  to the `gbdev <https://github.com/gbdev>`__ organization. The
+  `rgbds.gbdev.io <https://rgbds.gbdev.io>`__ website serving documentation and
```

Comment:
I would take the chance to split those two. The website was first made public on 17/5/2022, it didn't come with the move to the org.

Reply:
Thanks, I've done so.